backend/src/api/attendees.js

```diff
@@ -36,10 +37,13 @@ router.post(
   '/',
   errorWrap(async (req, res) => {
     const data = req.body
+    const candidate = await Candidate.findOne({ email: data.email })
+    const candidateId = candidate ? candidate._id : null
     const newAttendee = new Attendee({
       name: data.name,
       email: data.email,
-      year: data.year
+      year: data.year,
+      candidateId
```

Comment:
do you have to do `candidateId: candidateId` or is it filled in automatically by mongoose?

Reply:
Yeah, with javascript this is equivalent to putting `candidateId: candidateId`! It's not a mongo thing, just a javascript thing.

Reply:
javascript is wild